### PR TITLE
Enable GCC_NO_COMMON_BLOCKS.

### PR DESCRIPTION
**Pull request by calebd**

### CODE REVIEW

File: Base/Common.xcconfig [hunk 1]
```diff
@@ -172,3 +172,6 @@ RUN_CLANG_STATIC_ANALYZER = YES
 
 // Don't treat unknown warnings as errors, and disable GCC compatibility warnings and unused static const variable warnings
 WARNING_CFLAGS = -Wno-error=unknown-warning-option -Wno-gcc-compat -Wno-unused-const-variable -Wno-nullability-completeness
+
+// Enable no common blocks build setting
```
Comment (NachoSoto) [inline Base/Common.xcconfig]: Could you explain why we're adding this (the warning) and add a brief description of what the setting does?

**Comment by NachoSoto** [comment]
Sounds good then! :shipit:

File: Base/Common.xcconfig (hunk 1 above)
Comment (calebd) [inline Base/Common.xcconfig]: Yeah... I need to figure out exactly what this build setting does haha.

Comment (NachoSoto) [inline Base/Common.xcconfig]: I read the doc and I was confused TBH :P

**Comment by calebd** [comment]
Updated! I think that makes a little more sense.

File: Base/Common.xcconfig [hunk 2]
```diff
@@ -172,3 +172,8 @@ RUN_CLANG_STATIC_ANALYZER = YES
 
 // Don't treat unknown warnings as errors, and disable GCC compatibility warnings and unused static const variable warnings
 WARNING_CFLAGS = -Wno-error=unknown-warning-option -Wno-gcc-compat -Wno-unused-const-variable -Wno-nullability-completeness
+
+// This setting is on for new projects as of Xcode ~6.3, though it is still not
+// the default. It warns if the same variable is declared in two binaries that
```
Comment (NachoSoto) [inline Base/Common.xcconfig]: Oooh yeah that makes sense.

**Comment by NachoSoto** [comment]
Thanks @calebd!

**Pull request merged**